### DISCUSSION TITLE
How to query data from different databases in the same service

### DISCUSSION THREAD

**Discussion by NickCc6** [abpframework/abp#16591] [Q&A]
My data is stored in different databases, and I need to query and return them in the Service

![image](https://github.com/abpframework/abp/assets/42535452/de06429f-63a8-4967-9586-8d4e51bf0e4a)

**Comment by berkansasmaz** [comment]
There is an article written on the subject:
- https://community.abp.io/posts/configuring-multiple-dbcontexts-in-an-abp-framework-project-uoz5is3o

**Reply by NickCc6** [reply]
> There is an article written on the subject:
> 
> * https://community.abp.io/posts/configuring-multiple-dbcontexts-in-an-abp-framework-project-uoz5is3o

Thank you very much for providing the solution. I have learned from the article that it seems to be using the Custom Repository Methods. This method can also be achieved

But I want to know if it's like multi tenant
using (CurrentTenant.Change(tenantId))
{

}

For example:
using (_repository.change(connectionStringName))
 {
      //query data
 }
